src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "products")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private int price;
+    @OneToOne
+    private Category category;
```

Comment:
one product can have one category - ok!
how many products can be in one category?